lib/money/money.rb

```diff
@@ -114,10 +114,13 @@ def -@
   end
 
   def <=>(other)
+    return unless other.respond_to?(:to_money)
```

Comment:
can we remove this line? I'm worried people are using this for `money == 0` which will now give false even when money is empty